source/jormungandr/jormungandr/default_settings.py

```diff
@@ -109,3 +109,5 @@
     
 GRAPHICAL_ISOCHRONE = False
 HEAT_MAP = False
+
+PATCH_WITH_GEVENT_SOCKET = False
```

Comment:
Hmm, here is the tricky thing:

For kraken, with/without the patch, parallelization is always activated
For valalla, without the patch,  parallelization is not (always) activated